### PR TITLE
Source Close.com: add human readable titles to `api-key` and `start_d…

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What
- Added `api-key` title: `API Key`
- Added `start_date` title: `Replication Start Date`

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-close-com test report (commit [`ba1357420f`](https://github.com/airbytehq/airbyte/pull/27950/commits/ba1357420f550732cdb1f05c422727c6be5b2487)) - ✅

⏲️  Total pipeline duration: 06mn45s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-close-com/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-close-com docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/christograb-close-com-title-fix/1688591852/ba1357420f550732cdb1f05c422727c6be5b2487/source-close-com/0.4.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-close-com test
```

**Comment by octavia-squidington-iii** [comment]
##  source-close-com test report (commit [`ba1357420f`](https://github.com/airbytehq/airbyte/pull/27950/commits/ba1357420f550732cdb1f05c422727c6be5b2487)) - ✅

⏲️  Total pipeline duration: 05mn45s 

| Step                                                                    | Result   |
|:------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-close-com/metadata.yaml | ✅        |
| Connector version semver check                                          | ✅        |
| Connector version increment check                                       | ✅        |
| QA checks                                                               | ✅        |
| Code format checks                                                      | ✅        |
| Connector package install                                               | ✅        |
| Build source-close-com docker image for platform linux/x86_64           | ✅        |
| Acceptance tests                                                        | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/christograb-close-com-title-fix/1688597190/ba1357420f550732cdb1f05c422727c6be5b2487/source-close-com/0.4.1/output.html)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-close-com test
```

File: airbyte-integrations/connectors/source-close-com/source_close_com/spec.json [hunk 1]
```diff
@@ -8,11 +8,13 @@
...
       "api_key": {
+        "title": "API Key",
         "type": "string",
         "description": "Close.com API key (usually starts with 'api_'; find yours <a href=\"https://app.close.com/settings/api/\">here</a>).",
         "airbyte_secret": true
       },
       "start_date": {
+        "title": "Replication Start Date",
         "type": "string",
         "description": "The start date to sync data. Leave blank for full sync. Format: YYYY-MM-DD.",
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-close-com/source_close_com/spec.json:19]:
```suggestion
        "description": "The start date to sync data; all data after this date will be replicated. Leave blank to retrieve all the data available in the account. Format: YYYY-MM-DD.",
```

**Review by sherifnada — APPROVED** [review]
Nice!

**Pull request merged**